ingeniamotion/information.py

```diff
@@ -132,3 +147,156 @@ def register_exists(
         """
         drive = self.mc.servos[servo]
         return register in drive.dictionary.registers(axis)
+
+    def get_product_name(self, alias: str = DEFAULT_SERVO) -> str:
+        """Get the product name of the drive.
+
+        Args:
+            alias: alias of the servo.
+        Returns:
+            If it exists for example: "EVE-NET-E", "CAP-NET-E", etc.
+        """
+        drive = self.mc.servos[alias]
+        product_name = drive.dictionary.part_number
+        return f"{product_name}"
+
+    def get_target(self, alias: str = DEFAULT_SERVO) -> Union[int, str]:
```

Comment:
Maybe this function should be split in 3, for canopen communication, ecat, and ethernet

Reply:
Done